nxapi/nxtool.py

```diff
@@ -160,6 +156,14 @@ def get_filter(arg_filter):
     use_ssl = False
     
 es = elasticsearch.Elasticsearch(cfg.cfg["elastic"]["host"], use_ssl=use_ssl)
+# Get ES version from the client and avail it at cfg
+es_version =  es.info()['version'].get('number', None)
+if es_version is not None:
+    cfg.cfg["elastic"]["version"] = es_version.split(".")[0]
```

Comment:
Great ! Thank you !

A little problem: v5 is not handled in nxtransform.py (search for "Unspecified ES version"), so a "./nxtool.py -c nxapi.json  -x --colors" fail with "Unknown / Unspecified ES version in nxapi.json : 5" :-)